### PR TITLE
docker-cloud 1.0.1

### PR DESCRIPTION
**Pull request by tifayuki**

### CODE REVIEW

File: Library/Formula/docker-cloud.rb [hunk 1]
```diff
@@ -0,0 +1,78 @@
+class DockerCloud < Formula
+  desc "Docker Cloud - BUILD SHIP & RUN, ANY APP, ANYWHERE"
```
Comment (dunn) [inline Library/Formula/docker-cloud.rb]: No all-caps, and remove the application name from the description, thanks!

**Comment by tifayuki** [comment]
@dunn I have updated the description. Can you please review and merge it? 
Thank you

**Comment by allansun** [comment]
+1, wish to have the PR merged as soon as possible.

**Comment by dunn** [comment]
Thank you for your contribution! Merged in 741842586a22f838f2d3b6d97de8a52beb1836c9.

**Pull request closed without merging**